Source/WebCore/rendering/updating/RenderTreeBuilder.cpp

```diff
@@ -457,7 +457,8 @@ void RenderTreeBuilder::attachToRenderElementInternal(RenderElement& parent, Ren
         cache->childrenChanged(&parent, newChild);
 
     if (parent.hasOutlineAutoAncestor() || parent.outlineStyleForRepaint().outlineStyleIsAuto() == OutlineIsAuto::On)
-        newChild->setHasOutlineAutoAncestor();
+        if (!is<RenderMultiColumnSet>(newChild->previousSibling())) 
+            newChild->setHasOutlineAutoAncestor();
```

Comment:
is this because in case of "being moved out" we use the incorrect parent to decide whether this bit should be set?

Reply:
We decide whether this bit should be set based on the new parent it is being moved to which I don't think is the correct thing to do when moving out a spanner

Reply:
could you clarify that in the commit message, please?